HHS/Head-Start-TTADP
# [PROD] - Filter validation, update known issues, add backend "createdAt" scope for ARs

## Description of change
- Validation messaging and behavior added to the filter menu panel. (500)
- The need to press apply from individual buttons was removed (500)
- Bugs with updating the filters were fixed (516)
- A backend scope was added to filter activity reports by createdAt (527)
- Yarn audit's known vulnerability file was updated
- Duplicate 'Pregnant Women' removed from target pop constant

## How to test
- Use the filter panel on the recipient record page. Check that the filters still work as expected, and that the validation messaging appears appropriately, and styled as expected. Note that you no longer have to click apply on the individual dropdowns, and that the date picker has a border and is a little harder to accidentally close.
- For the backend scope and yarn audit vulnerability, the only test is that Circle CI successfully completes
- Verify 'Pregnant Women' only shows up in the target populations dropdown once.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-500
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-516
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-527

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
